tests/dhcp_relay/test_dhcp_pkt_fwd.py

```diff
@@ -0,0 +1,329 @@
+import logging
+import random
+import ipaddr
+import ipaddress
+import pytest
+
+import ptf.testutils as testutils
+import ptf.packet as scapy
+
+from ptf.mask import Mask
+from socket import INADDR_ANY
+
+pytestmark = [
+    pytest.mark.topology("t1")
+]
+
+logger = logging.getLogger(__name__)
+
+class DhcpPktFwdBase:
+    """
+    Base class for DHCP packet forwarding test. The test ensure that DHCP packets are going through T1 device.
+    """
+    LEASE_TIME = 86400
+    DHCP_PKT_BOOTP_MIN_LEN = 300
+    DHCP_CLIENT = {
+        "mac": "00:11:22:33:44:55",
+        "ip": "10.10.10.1",
+        "subnet": "255.255.255.0",
+        "port": 68,
+    }
+    DHCP_RELAY = {
+        "mac": "22:33:44:55:00:11",
+        "ip": "20.20.20.1",
+        "loopback": "10.0.0.33",
+    }
+    DHCP_SERVER = {
+        "mac": "44:55:00:11:22:33",
+        "ip": "30.30.30.30",
+        "port": 67,
+    }
+
+    def __getPortLagsAndPeerIp(self, duthost, testPort):
```

Comment:
I'm a little confused about this method. If the topo has no LAGs (like t1) or we select a port that is not a LAG, then does this just return the port?

Reply:
Yes, if there are not lag members, the testPort will be returned. If the testPort is part of LAG membership, the testPort and LAG members will be returned in this case.